File: 단계별로_풀어보기/1. 입출력과 사칙연산/9. 10430.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString().split(' ')

let [A, B, C] = inputs

A = parseInt(A)
B = parseInt(B)
C = parseInt(C)

// prettier-ignore
const resultsArr = [
  (A + B) % C,
  ((A % C) + (B % C)) % C,
  (A * B) % C,
  ((A % C) * (B % C)) % C
]

process.stdout.write(resultsArr.join('\n'))

File: 1. 단계별로_풀어보기/1~9/1. 입출력과 사칙연산/9. 10430.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

A, B, C = map(int, input().split(' '))

output(f"{(A + B) % C}\n")
output(f"{((A % C) + (B % C)) % C}\n")
output(f"{(A * B) % C}\n")
output(f"{((A % C) * (B % C)) % C}")